docs/design/validation.md

```diff
@@ -0,0 +1,35 @@
+# Validation
+
+## Problem
+Kubernetes has restrictions for the possible values of resources. Therefore, Dashboard must
+validate user input.
+
+Please note: some input fields of Dashboard are used to create multiple resources in Kubernetes.
```

Comment:
Basically, this is correct. However, I have not investigated if the value restrictions are aligned or not. But, in any case one should be aware of potential problems.